### ISSUE TITLE
Version 2.17.9 is not installable

### ISSUE THREAD

**Issue by eribertomota** [oetiker/mrtg#84]
Hi @oetiker 

Thanks for release the 2.17.9 version. However, this version is not installable because the file doc/mrtglib.3 does not exist (but there is the old, no longer needed,  mrtglib.1). See below:

```
# make install
[...]
for x in ./doc/*.3; do \
  /usr/bin/install -c -m 644 $x /opt/mrtg-2/share/man/man3; done
/usr/bin/install: cannot stat './doc/*.3': No such file or directory
make: *** [Makefile:70: install] Error 1
```
As a workaround, I ran `make doc` before `make install`.

Cheers,

Eriberto

**Comment by oetiker** [comment]
thanks ... just released 2.17.10 ... please test

**Comment by eribertomota** [comment]
Working like a Swiss watch. Congratulations and thanks!
